File: src/main/java/ru/mdashlw/hypixel/ranked/reply/impl/PlayerReply.java
package ru.mdashlw.hypixel.ranked.reply.impl;

import com.fasterxml.jackson.annotation.JsonProperty;
import ru.mdashlw.hypixel.ranked.entities.RankedPlayer;
import ru.mdashlw.hypixel.ranked.reply.Reply;

public final class PlayerReply extends Reply<RankedPlayer> {
    @JsonProperty
    private RankedPlayer player;

    @Override
    public RankedPlayer getValue() {
        return this.player;
    }
}

File: src/main/kotlin/ru/mdashlw/hypixel/api/ranked/reply/impl/PlayerReply.kt
package ru.mdashlw.hypixel.api.ranked.reply.impl

import ru.mdashlw.hypixel.api.ranked.entities.Player
import ru.mdashlw.hypixel.api.ranked.reply.Reply

class PlayerReply(
    success: Boolean,
    cause: String?,
    player: Player?
) : Reply<Player>(success, cause, player)
